tools/scoring/score/parse_config.py

```diff
@@ -149,6 +149,8 @@ def retrieve_reporting_translations(
     for solution_entity in solution_entities.values():
       if solution_entity.cloud_device_id is None:
         continue  # as this is not a reporting device
+      if not Dimension.is_entity_canonical(solution_entity):
```

Comment:
is this for virtual entities?

Reply:
No, it's just an explicit check that simple dimensions are scoring only _canonical_ reporting entities. The complex dimensions invoke it independently.